### PR TITLE
Bugs/frontend/theme/views

### PR DESCRIPTION
**Pull request by danpolanco**
Addresses issues #61 #51 #55 #47.

### CODE REVIEW

**Comment by itzg** [comment]
Other than the login page all of these changes look great.

As for the login, override the `margin-bottom` for the `.login-header` to be 0, so that it sits against the white part of the login form.

image

It turns out that'll help fix the failed login rendering:

image

It also looks like the `div.login-footer` also needs a `col-xs-12`, but not sure it that'll help the weird vertical extent it's showing here:

image

File: src/main/resources/static/ng-bits/current-container.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 </div>
 
 <div class="panel-body row">
-    <div class="col-sm-10 row">
-        <div class="field col-sm-6"><label>Type</label> <span>{{c.type | serverTypeLabel}}</span></div>
-        <div class="field col-sm-6"><label>Version</label> <span>{{c.version}}</span></div>
-        <div class="field col-sm-6" ng-show="hasConnectInfo(c)">
+    <div class="col-xs-10 row">
```
Comment (itzg) [inline src/main/resources/static/ng-bits/current-container.html:8]: Actually `xs` probably shouldn't be used here. That one is special in that it overrides the responsive, collapsing behavior of Bootstrap grids:

http://getbootstrap.com/css/#grid-options

however, I say this without having yet tested these latest changes with an iPhone.

**Comment by danpolanco** [comment]
I believe it's ready for merge.

**Comment by danpolanco** [comment]
screen shot 2016-01-24 at 3 53 17 pm

Could always use more tweaking, but I think it looks good for now :grinning:

**Pull request merged**